go.mod

```diff
@@ -0,0 +1,23 @@
+module github.com/ONSdigital/dp-frontend-router
+
+go 1.12
+
+require (
+	github.com/ONSdigital/dp-api-clients-go v1.0.0
+	github.com/ONSdigital/dp-healthcheck v0.0.0-20200131122546-9db6d3f0494e
+	github.com/ONSdigital/go-ns v0.0.0-20191104121206-f144c4ec2e58
+	github.com/ONSdigital/log.go v0.0.0-20200128163509-ddbd03a854d6
```

Comment:
And check you have the latest log.go lib please - some new fixes were applied recently thanks